src/api/mock.ts

```diff
@@ -54,6 +54,11 @@ export interface MockBase {
      */
     simulateDelay(value: boolean): void;
 
+    /**
+     * Enables or disables logging requests without matching handlers.
+     */
+    logUnhandledRequests(value: boolean): void;
```

Comment:
Nice feature, but does it have to be configurable? Shouldn't this always be turned on?

Reply:
Unit tests often have many unnecessary and unhandled requests. It would take a lot of extra code to get rid of all logged requests then